fe/fe-core/src/main/java/org/apache/doris/catalog/DataProperty.java

```diff
@@ -75,11 +94,20 @@ public static DataProperty read(DataInput in) throws IOException {
     public void write(DataOutput out) throws IOException {
```

Comment:
Use gson to do serialize and deserialize

Reply:
`DataProperty` is a already exist class. I can not change the serialize/deserialize to GSON since the backward compatible